interface/drugs/add_edit_drug.php

```diff
@@ -14,6 +14,8 @@
  require_once("drugs.inc.php");
  require_once("$srcdir/options.inc.php");
 
+$drugService = new \services\DrugService();
```

Comment:
then this will be:
$drugService = new DrugService();

Reply:
disregard above comment for now. I am in process of converting this namespace, so can do this in the future.